src/AltinnRepositories/gitea-data/gitea/templates/base/head.tmpl

```diff
@@ -141,23 +141,27 @@
 			<div class="ui top secondary stackable main menu following bar light">
 				<div class="ui container" id="navbar">
 					<div class="item brand" style="justify-content: space-between;">
-						<a href="{{AppSubUrl}}/" href="{{AppSubUrl}}/">
-							<img class="ui mini image" src="{{AppSubUrl}}/img/altinn_logo_header.png">
+						<a href="/" title="Altinn Studio {{.i18n.Tr "home"}}">
+							<img class="ui mini image" src="{{AppSubUrl}}/img/favicon.png" alt="Altinn logo">
```

Comment:
CDN?

Reply:
No, this is not used across applications.